install_files/ansible-base/roles/ossec_server/tasks/mon_configure_custom_cert.yml

```diff
@@ -0,0 +1,39 @@
+---
+  # Create a separate directory for storing a custom
+  # cert file, if declared. A separate dir is necessary
+  # because otherwise removing a custom cert from the
+  # Ansible config would NOT result in the custom cert
+  # being removed from the /etc/ssl/certs.
+- name: ensure custom certificate directory exists (if necessary)
+  file:
+    path: "{{ smtp_relay_cert_override_dir }}"
+    state: directory
+    mode: "0755"
```

Comment:
FWIW, from the Postfix TLS docs:
> The file is opened (with root privileges) before Postfix enters the optional chroot jail and so need not be accessible from inside the chroot jail.

So in theory you could tighten the permissions to `0700` without breaking functionality. However, that change isn't necessary because the contents of the _certificate_ file can be world-readable without harm.